concent-vm/install-concent.yml

```diff
@@ -0,0 +1,90 @@
+- hosts:
+    - vagrant
+  vars_files:
+    - ../containers/versions.yml
+    - ../concent-builder/repositories.yml
+    - vars.yml
+  tasks:
```

Comment:
wouldnt be better to put ansible logic in separate roles? It is covered in best practicies: https://docs.ansible.com/ansible/latest/user_guide/playbooks_best_practices.html 

"One crucial way to organize your playbook content is Ansible’s “roles” organization feature, which is documented as part of the main playbooks page. You should take the time to read and understand the roles documentation"

Ideally, playbook should be just mapping between role and a host. Each role can have different set of vars, files and templates. This modular design is easier to maintain, reuse and understand.

Roles can include:
-PosgreSQL (install and migrate data)
-nginx (install and setup service)
-prepare directories and users
-system level dependencies
-build and run concent
-run tests

Reply:
I fully agree. I have even suggested splitting it in the pull request this one builds upon (https://github.com/golemfactory/concent-deployment/pull/159#discussion_r204779817). It was less of a problem when we had only a couple playbooks but now they're starting to get repetitive so it's a good idea to split them into smaller parts.

I suggested just putting tasks into smaller playbooks and importing them to keep things simple for now. But roles are fine too.

@bartoszbetka: @michalpiasecki1 has lots of experience with Ansible so please talk with him if you have any questions about how to split it.

BTW, @michalpiasecki1 if you want to create a bullet list in Markdown you have to put a space between `-` and the text. I have edited your comment to show that. See how it looks now.